### ISSUE TITLE
Feature request: prevent autofocus on focusable element and skip to next focusable element with a data- attribute

### ISSUE THREAD

**Issue by alex-e-leon** [theKashey/react-focus-lock#212]
In my application I have tooltips like this:

<img width="34" alt="Screen Shot 2022-03-18 at 10 04 56 am" src="https://user-images.githubusercontent.com/7065873/159049603-ccffdc34-4604-451f-ac97-70992fb15eff.png">

Which according to [WCAG](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) should be focusable - however having a tooltip focused and shown whenever a dialog is opened is a pretty strange behaviour.

It would be nice if I could "blacklist" the element, possibly by adding a data-attribute (maybe something like `data-no-autofocus`, so that focus-trap would focus the next element instead when a dialog is opened.

**Comment by theKashey** [comment]
A very good suggestion. I like it.

**Comment by theKashey** [comment]
Now(react-focus-lock@2.9.0) one can control autofocus behavior by setting `data-no-autofocus` on an element or container - https://github.com/theKashey/focus-lock#declarative-control
That should let you control autofocus behavior as desired.

**Comment by alex-e-leon** [comment]
Thanks @theKashey - I'll test it out and report, but this looks great : )

**Comment by alex-e-leon** [comment]
Tested locally, and seems to work great.

**Comment by theKashey** [comment]
:yay:. thank you for pointing on the idea as well as the implementation. From some point of view - that's all your doing.
